src/test/java/com/microsoft/sqlserver/jdbc/TestUtils.java

```diff
@@ -71,52 +73,30 @@
     private static Boolean isAzureDW = null;
     private static Boolean isAzureMI = null;
 
-    /*
-     * SERVERPROPERTY('EngineEdition') can be used to determine whether the db server is SQL Azure.
-     * It should return 6 for SQL Azure DW. This is more reliable than @@version or
-     * serverproperty('edition').
-     * Reference: http://msdn.microsoft.com/en-us/library/ee336261.aspx
+    /**
+     * Check if connected to Azure server.
```

Comment:
Checks if the connection is established to an Azure server. Please make similar changes to below.